### DISCUSSION TITLE
Is there any parameter setting inside OpenROAD to speed up DRT operation?

### DISCUSSION THREAD

**Discussion by YiHaoHuangGDUT** [The-OpenROAD-Project/OpenROAD-flow-scripts#1607] [Q&A] [answered]
There is a teaching project that requires demonstrating process collaborative design to students, so I need to run the entire process frequently, but the DRT step is too time-consuming, and I want to speed it up. The platform used is ASAP7 and the design is swerv. What can I do?

**Comment by maliberty** [comment]
How many cpus do you have and how long does it take?  You could stop after an earlier number of iterations and leave some DRCs.  You could stop at global routing and evaluate timing there and save final routing for overnight runs.

**Reply by rovinski** [reply, marked as answer]
DRT scales quite well with cores so you could simply throw more CPUs at it 🤷‍♂️
You could also use a different technology because ASAP7 takes longer to route than other technologies.
You could also pick a smaller design so it runs faster overall.

**Reply by YiHaoHuangGDUT** [reply]
Thank you! I previously deployed OpenROAD on my personal computer, which had lower hardware specifications with a single CPU and 16GB of RAM. Now, after deploying it on a server, the speed has significantly improved.
